membership-attribute-service/test/integration/AccountDetailsFromZuoraIntegrationTest.scala

```diff
@@ -23,14 +24,15 @@ class AccountDetailsFromZuoraIntegrationTest extends Specification {
       val result = Await
         .result(
           createTouchpointComponents.accountDetailsFromZuora
-            .fetch("200152344", AccountHelpers.NoFilter)
+            .fetch("200264404", AccountHelpers.NoFilter)
             .run
             .map { list =>
               println(s"Fetched this list: $list")
               list
             },
           Duration.Inf,
         )
+      result.map(_.foreach(accountDetails => println(Json.prettyPrint(accountDetails.toJson))))
```

Comment:
Assuming we don't want this debugging?

Reply:
This is an integration test and it's set up to be ignored, so the only time I run it is when I'm debugging stuff and then I usually do want the logging